videoprocess/process_sharpness.cfg.template

```diff
@@ -0,0 +1,37 @@
+# Configuration information for video process test case.
+#    This application will firstly load yuv frames to one type of surface(NV12/YV12/I420)
+#  you require. After video processing, the processed content (NV12/YV12/I420 surface)
+#  will be stored to frames(yv12 format in file).
+#    Supported features include: sharpening, implicit format conversion(NV12<->YV12<->I420). Each
+# time only one kind of processing will be executed in test application. Although libva supports
+# multiple filters execution in one time. you can modify this configuration file to set the
+# filter and the corresponding parameters.
+
+#1.Source YUV(RGB) file information
+#SRC_FILE_NAME:    /root/clips/YUV/bus_cif.yv12
+SRC_FILE_NAME: ./flowersky_352x288.nv12
+SRC_FRAME_WIDTH: 352 
+SRC_FRAME_HEIGHT: 288
+SRC_FRAME_FORMAT: NV12
+
+#Note .yv12 files are in YV12 format and .yuv used above for STDE is in I420 format
+SRC_FILE_FORMAT: NV12
+
+#2.Destination YUV(RGB) file information
+DST_FILE_NAME:    ./flowersky_352x288_writer352x288.nv12
+DST_FRAME_WIDTH:  352
+DST_FRAME_HEIGHT: 288
+DST_FRAME_FORMAT: NV12
+
+DST_FILE_FORMAT: NV12
+
+#3.How many frames to be processed
+FRAME_SUM: 3
```

Comment:
Suggest to unify the frame no, like 5 as previous chromasitting application.

Reply:
fixed, change to 5